backend/utils/utility.py

```diff
@@ -0,0 +1,133 @@
+import requests
+from utils.db_handler import DB
+
+
+async def get_rooms(member_id, org_id):
+    """Get the rooms a user is in
+    Args:
+        member_id (str): The user id
+    Returns:
+        [List]: [description]
+    """
+    DB.organization_id = org_id
+    query = {"room_members": member_id}
+    options = {"sort": {"created_at": -1}}
+    response = await DB.read("rooms", query=query, options=options)
+
+    if response and "status_code" not in response:
+        return response
+    return []
+
+
+async def get_all_organization_members(org_id: str):
```

Comment:
the get_all members and get member function could be placed directly under the db_handler class or better still you could create another class to hold extra helper functions. rather than making them separate functions